### PR TITLE
OnFido Create Applicant | Template Review | OKTA-380518

### PR DESCRIPTION
**Pull request by oleksandrkysil-okta**
Jira ticket:
- https://oktainc.atlassian.net/browse/OKTA-380518
Description:
- Template Review
Note:
- There is no youtube video link, but readme file could be checked without it.

### CODE REVIEW

File: workflows/onfido_create_applicant/workflow.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "onfido_create_applicant",
+  "title": "Onfido Create Applicant",
+  "description": "Using the Okta User Created Event, this workflow will create an Onfido Applicant and save the applicant ID into the user's profile in Okta.",
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/workflow.json:4]: Please replace the description with the following edited version:

With the User Created event card for the Okta connector, this flow creates an Onfido applicant and saves the applicant ID in the user’s Okta profile.

File: workflows/onfido_create_applicant/readme.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## Onfido Create Applicant
+
+### Overview
+
+This workflow creates an Onfido applicant from the Okta Create User event for use in Identity Verification. The Okta user profile is updated with the Onfido Applicant ID.
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:5]: Please replace this opening paragraph with the following:

For identity verification, this flow creates an Onfido applicant using a User Created event card for the Okta connector and saves the applicant ID in the user’s Okta profile.

File: workflows/onfido_create_applicant/readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Creating an applicant in Onfido at Okta user creation allows organizations to ensure that each newly registering user has an Onfido Applicant ID that can be used for Identity Verification Checks. This applicant ID can be reused for other Onfido functionality such as facial biometric re-checks, initializing the Onfido SDK, and collecting report information for a given check on a specific user.
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:7]: Please replace this paragraph with the following:

When you create an application in Onfido as part of the Okta user creation process, your organization ensures that each newly registered user has an Onside application ID that can be used for identity verification checks. This applicant ID can be reused for other Onfido features such as facial biometric re-checks, initializing the Onfido SDK, and collecting report information for a check on a specific user.

File: workflows/onfido_create_applicant/readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+### Before you get Started / Prerequisites
+
+Before you get started, you will need:
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:11]: Remove this sentence

File: workflows/onfido_create_applicant/readme.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+### Testing this Flow
+
+1. Create a user in Okta
+2. Verify that the `[Onfido] UserEvent_Create Applicant` workflow executes successfully and the user profile attribute `onfidoApplicantId` is updated with a value. 
+
+
+### Limitations & Known Issues
+
+* OnFido applicants are created for all Okta Users that are created. Additional conditional logic can be added in the workflow to create OnFido applicants based on user status, group membership, etc.
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:43]: replace "workflow" with "flow"

File: workflows/onfido_create_applicant/readme.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+### Testing this Flow
+
+1. Create a user in Okta
+2. Verify that the `[Onfido] UserEvent_Create Applicant` workflow executes successfully and the user profile attribute `onfidoApplicantId` is updated with a value. 
+
+
+### Limitations & Known Issues
+
+* OnFido applicants are created for all Okta Users that are created. Additional conditional logic can be added in the workflow to create OnFido applicants based on user status, group membership, etc. 
+* Keep in mind the [Okta Workflows System Limits](https://help.okta.com/en/prod/Content/Topics/Workflows/workflows-system-limits.htm) of Event Hook and Okta API rate limits.
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:44]: Replace with:

Refer to the [Okta Workflows System Limits](https://help.okta.com/en/prod/Content/Topics/Workflows/workflows-system-limits.htm) for event hooks and Okta API rate limits.

File: workflows/onfido_create_applicant/readme.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+* Onfido also has rate limits for API calls: [Onfido Rate Limit](https://documentation.onfido.com/#rate-limits) that should be taken into account when designing workflows.
```
Comment (markmoussa-okta) [inline workflows/onfido_create_applicant/readme.md:45]: replace "workflows" with "flows"

File: workflows/onfido_create_applicant/readme.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+## Onfido Create Applicant
+
+### Overview
+
+This workflow creates an Onfido applicant from the Okta Create User event for use in Identity Verification. The Okta user profile is updated with the Onfido Applicant ID.
+
+Creating an applicant in Onfido at Okta user creation allows organizations to ensure that each newly registering user has an Onfido Applicant ID that can be used for Identity Verification Checks. This applicant ID can be reused for other Onfido functionality such as facial biometric re-checks, initializing the Onfido SDK, and collecting report information for a given check on a specific user.
+
+### Before you get Started / Prerequisites
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:9]: Remove "Before you get Started /"

File: workflows/onfido_create_applicant/readme.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+## Onfido Create Applicant
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:1]: @oleksandrkysil-okta I think we want a single # here, and then please adjust all other sections.

File: workflows/onfido_create_applicant/readme.md (hunk 8 above)
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:9]: I think we want ## not ###

File: workflows/onfido_create_applicant/readme.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+### Overview
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:3]: I think we want ## not ###

File: workflows/onfido_create_applicant/readme.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+### Before you get Started / Prerequisites
+
+Before you get started, you will need:
+
+* Access to an Okta tenant with Okta Workflows enabled for your org
+* An Onfido instance with an API Token generated for use in this workflow (see: https://documentation.onfido.com/#token-authentication). Choose `Sandbox` or `Live` as appropriate for your Onfido instance. 
+* A custom Okta Universal Directory profile attribute named: `onfidoApplicantId` of type `string`
+
+
+### Setup Steps
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:18]: I think we want ## not ###

File: workflows/onfido_create_applicant/readme.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+2. Open the `[Onfido] UserEvent_Create Applicant` flow and modify the following:
+
+- On the `Text Compose` card with the Onfido API URL, enter the correct Onfido URL for your region.
+- Choose the Okta connection for the `Okta Read User` and `Okta Update User` cards
+- Choose the `Onfido API` HTTP Connection for the `HTTP Post` card
+ 
+3. Save and Turn on the `[Onfido] UserEvent_Create Applicant` flow.
+
+### Testing this Flow
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:35]: I think we want ## not ###

File: workflows/onfido_create_applicant/readme.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Create a user in Okta
+2. Verify that the `[Onfido] UserEvent_Create Applicant` workflow executes successfully and the user profile attribute `onfidoApplicantId` is updated with a value. 
+
+
+### Limitations & Known Issues
```
Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:41]: I think we want ## not ###

Comment (davidroossien-okta) [inline workflows/onfido_create_applicant/readme.md:41]: I think we want ## not ###

**Review by davidroossien-okta — CHANGES_REQUESTED** [review]

File: workflows/onfido_create_applicant/readme.md (hunk 2 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: replaced

File: workflows/onfido_create_applicant/readme.md (hunk 3 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: updated

File: workflows/onfido_create_applicant/readme.md (hunk 4 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: removed

File: workflows/onfido_create_applicant/readme.md (hunk 5 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: replaced

File: workflows/onfido_create_applicant/readme.md (hunk 6 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: replaced

File: workflows/onfido_create_applicant/readme.md (hunk 8 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: removed

File: workflows/onfido_create_applicant/readme.md (hunk 9 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

File: workflows/onfido_create_applicant/readme.md (hunk 8 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

File: workflows/onfido_create_applicant/readme.md (hunk 10 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

File: workflows/onfido_create_applicant/readme.md (hunk 11 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

File: workflows/onfido_create_applicant/readme.md (hunk 12 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

File: workflows/onfido_create_applicant/readme.md (hunk 13 above)
Comment (oleksandrkysil-okta) [reply workflows/onfido_create_applicant/readme.md]: changed

**Review by davidroossien-okta — APPROVED** [review]

**Pull request merged**